src/directional_clustering/plotters/ply_plotter.py

```diff
@@ -0,0 +1,138 @@
+from numpy import zeros
+from numpy import asarray
+
+# plotter is based on Plotly (https://plotly.com/python/)
+import plotly.graph_objects as go
+import plotly.figure_factory as ff
+
+from compas.geometry import length_vector
+
+from directional_clustering.plotters import mesh_to_vertices_xyz
+from directional_clustering.plotters import lines_to_start_end_xyz
+from directional_clustering.plotters import lines_start_end_connect
+from directional_clustering.plotters import vectors_dict_to_array
+from directional_clustering.plotters import face_centroids
+from directional_clustering.plotters import line_sdl
+from directional_clustering.plotters import rgb_colors
+
+__all__ = [
+    "PlyPlotter"
+]
+
+# TODO: see decorator style
+
+class PlyPlotter(go.Figure):
+    """
+    PlyPlotter is a custom wrapper around a Plotly.py graph object plotter.
+    """
+
+    def __init__(self, *args):
+        super(PlyPlotter, self).__init__(*args)
+
+    def set_title(self, title):
+        """
+        Sets title of the plot and sets the aspect ratio to the data. The default aspect ratio is of a cube, which distorts the image.
+        """
+        self.update_layout(
+            title_text=title,
+            showlegend=False,
+            scene=dict(aspectmode='data'))
+
+    def draw_vector_field_array(self, mesh, vectors, color, uniform, scale, width=0.5):
```

Comment:
Suggestion: Changing the word `draw` for `plot`. Seems like a better descriptor.
Is the `vectors` argument a `VectorField`? If so, I suggest we change the argument name to `vector_field`, and the function name to `plot_vector_field`. Would be clearer? 😄

Reply:
Sure! incorporated changes to other "draw"'s too